web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import ProTable, {
+  ProTableProps,
+} from "@reearth-cms/components/atoms/ProTable";
+import { Asset } from "@reearth-cms/components/molecules/Asset/asset.type";
+import { ConfigProvider } from "antd";
+import { Locale } from "antd/lib/locale-provider";
```

Comment:
Is this a separate locale/intl context to react-i18next? Or am i misunderstanding the purpose of this configProvider and Locale?

Reply:
I only added this because the default language of AntD ProTable was Chinese and I wanted change it to English.
Please let me know if you have any suggestion.

Reply:
I wrapped the main component with "<ConfigProvider locale={enUSIntl}>" to apply locale for all antd components globally.